x-pack/plugins/security_solution/public/resolver/types.ts

```diff
@@ -42,6 +46,14 @@ export interface ResolverUIState {
    * The ID attribute of the resolver's currently selected descendant.
    */
   readonly selectedDescendantId: string | null;
+  /**
+   * The entity_id of the process for the resolver's currently selected descendant.
+   */
+  readonly processEntityIdOfSelectedDescendant: string | null;
```

Comment:
what does 'descendant' mean in this case?
I no longer understand resolvers model for selecting things.